File: TCloudFileSync/Backend/src/TCloudFileSync.Angular/ClientApp/src/app/domain/entity/rotina/rotina.ts
export class Rotina{
    public id?: number;
    public ativo?: any;
    public caminhoArquivoLocal?: string;
    public caminhoArquivoNuvem?: string;
    public tempoParaIniciarProximoEnvio?: number;
    public deveApagarArquivoOrigem?: any;
    public fluxoLocalParaNuvem?: any;
    public idConfiguracaoSftp?: number;
}

File: TCloudFileSync/Backend/src/TCloudFileSync.Infra/Entidade/Rotina.cs
namespace TCloudFileSync.Infra.Entidade
{
    public class Rotina : Entidade
    {
        public string Ativo { get; set; }
        public string CaminhoArquivoLocal { get; set; }
        public string CaminhoArquivoNuvem { get; set; }
        public int TempoParaIniciarProximoEnvio { get; set; }
        public string DeveApagarArquivoOrigem { get; set; }
        public string FluxoLocalParaNuvem { get; set; }

        public int IdConfiguracaoSftp { get; set; }
        public ConfiguracaoSftp ConfiguracaoSftp { get; set; }
    }
}
